packages/iTwinUI-react/src/core/Footer/Footer.tsx

```diff
@@ -28,6 +31,11 @@ export type FooterProps = {
    * Provide localized strings.
    */
   translatedTitles?: TitleTranslations;
+  /**
+   * Custom footer content. If provided, it will render only what you passed.
+   * If function provided, it should accept an array of default footer elements and return `React.ReactNode`.
```

Comment:
Comment is out of date now

Reply:
Changed. Now mentioned `defaultFooterElements`